mkdocs.yml

```diff
@@ -23,6 +23,7 @@ nav:
       - Other Tools: community/other_tools.md
       - Code of Conduct: community/CODE_OF_CONDUCT.md
       - Team: community/team.md
+    -OHBM 2023 Tutorials: ohbm23_tutorials.md
```

Comment:
You just need to add a space after the dash and before OHBM for it to correctly catch the section!